CoreDataStack/Entity/UploadFile+CoreDataProperties.swift

```diff
@@ -8,18 +17,20 @@ extension UploadFile {
         return NSFetchRequest<UploadFile>(entityName: "UploadFile")
     }
 
-    @NSManaged public var fileType: Int64
-    @NSManaged public var fileSize: Double
-
-    @NSManaged public var name: String?
+    @NSManaged public var content: Data?
     @NSManaged public var createdAt: Date?
+    @NSManaged public var fileSize: Double
+    @NSManaged public var fileType: Int64
     @NSManaged public var id: String?
     @NSManaged public var key: String?
     @NSManaged public var landingTxID: String?
+    @NSManaged public var name: String?
     @NSManaged public var payloadTxID: String?
     @NSManaged public var provider: String?
+    @NSManaged public var mime: String?
 
-    @NSManaged public var content: Data?
 }
 
-extension UploadFile : Identifiable {}
+extension UploadFile : Identifiable {
+
```

Comment:
This change is unneeded.

Reply:
The file is generated by xcode,  I think no need to change the content, because the change will be override in next generation.